supabase/supabase
# Fix/side panel editor invalid date inputs

Addresses several recent reports of dates not working in the SidePanelEditor

ie: https://support.supabase.org/a/tickets/6380

I added and double checked that these fields work: 
- date
- time, 
- timetz
- timestamp
- timestamptz

**To test:** 
1. Create a new table
2. Add all of the above fields to it
3. Ensure a date/time/timestamp can be added successfully 

<img width="643" alt="CleanShot 2022-05-16 at 22 20 40@2x" src="https://user-images.githubusercontent.com/105593/168705208-67656d1c-ab2a-4f61-b8e6-fed08d19415d.png">


We should do a triple check before we merge this.
